templates/registration/login.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Login</h1>
+  {% if form.error %}
+    <p>Invalid credentials</p>
+  {% endif %}
+  <form action="{% url "login" %}" method="post">

```

Comment:
The form action URL `{% url "login" %}` should be verified to ensure it matches the URL pattern defined for the login view. Typically, this is handled by Django's built-in authentication system.